src/components/Quotes.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import QUOTE_API from "../const";
+
+const getQuote = async () => {
+  const response = await fetch(QUOTE_API);
+  const data = await response.json();
+  return data;
+};
+
+export default function Quotes() {
+  const [quote, setQuote] = useState(null);
+  const [loading, setLoading] = useState(true);
+
```

Comment:
Please consider adding an error state in the component so that when the use effect function returns an error promise in your case it sets the error state and shows the error to the user just as you showed the loading state.

You can get the error in the catch block of your promise.


```jsx
 if (hasError || Number(quotes.length) === 0)
    return <div>Something went wrong!</div>
```

```
const [hasError, setHasError] = useState(false)
```